pkg/assets/assets.go

```diff
@@ -17,6 +17,7 @@
 package assets
```

Comment:
I'm thinking about https://github.com/kinvolk/lokomotive/issues/795, I wonder if maybe new things should be added into `internal/assets` for example.

Reply:
I've commented on the issue. I wouldn't start moving things around before we have a clear picture of what we want to export. Right now there is nothing in our codebase which suggests that people can rely on the API of Lokomotive libraries so if people do that it's at their own risk as far as I'm concerned.